### PR TITLE
Avoid Failure when Rotating Endpoint with multiple certificates

### PR DESCRIPTION
**Pull request by hosseinsh**
Lemur currently gives up, when rotating an endpoint with multiple new certificates to replace the old one. 
In case of duplicate certificates replacing an expiring one, Lemur shall select the new certificate at index 0, basically the first available certificate.

### CODE REVIEW

File: lemur/certificates/cli.py [hunk 1]
```diff
@@ -244,41 +249,24 @@ def rotate(endpoint_name, new_certificate_name, old_certificate_name, message, c
                 current_app.logger.info(log_data)
 
         else:
+            # no certificate name or endpoint was provided, so we will now fetch all endpoints,
+            # which have are attached to a certificate that has been replaced
```
Comment (charhate) [inline lemur/certificates/cli.py:253]: `which have are attached` --> `which are attached`

File: lemur/certificates/cli.py [hunk 2]
```diff
@@ -244,41 +249,24 @@ def rotate(endpoint_name, new_certificate_name, old_certificate_name, message, c
...
-                    )
+                log_data["message"] = "Rotating endpoint from old to new cert"
+                if len(endpoint.certificate.replaced) > 1:
+                    log_data["message"] = f"Multiple replacement certificates found, going with the first one out of " \
+                                          f"{len(endpoint.certificate.replaced)}"
+
+                log_data["endpoint"] = endpoint.dnsname
+                log_data["certificate"] = endpoint.certificate.replaced[0].name
+                request_rotation(endpoint, endpoint.certificate.replaced[0], message, commit)
+                print(log_data)
```
Comment (charhate) [inline lemur/certificates/cli.py:265]: Is explicit `print(log_data)` given `current_app.logger.info(log_data)`

Comment (hosseinsh) [reply lemur/certificates/cli.py]: good point, we can skip them.

File: lemur/certificates/cli.py [hunk 3]
```diff
@@ -418,32 +406,27 @@ def rotate_region(endpoint_name, new_certificate_name, old_certificate_name, mes
...
+                log_data["certificate"] = endpoint.certificate.replaced[0].name
+                log_data["message"] = "Rotating all endpoints in region"
+                if len(endpoint.certificate.replaced) > 1:
+                    log_data["message"] = f"Multiple replacement certificates found, going with the first one out of " \
+                                          f"{len(endpoint.certificate.replaced)}"
+
+                request_rotation(endpoint, endpoint.certificate.replaced[0], message, commit)
+                current_app.logger.info(log_data)
 
                 metrics.send(
```
Comment (charhate) [inline lemur/certificates/cli.py:424]: Are the metrics missing in above if and elif?

Comment (hosseinsh) [reply lemur/certificates/cli.py:424]: no, they follow later, line 467

**Review by charhate — APPROVED** [review]

**Pull request merged**